manifests/b/BlizzardEntertainment/Battle/netSetup/1.18.5.3106/BlizzardEntertainment.Battle.netSetup.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: BlizzardEntertainment.Battle.netSetup
+PackageVersion: 1.18.5.3106
+Installers:
+- InstallerUrl: https://us.battle.net/download/getInstaller?os=win&installer=Battle.net-Setup.exe&id=undefined

```

Comment:
This file return a different hash every time it is downloaded.

Reply:
Yeah I just figured that out. Apologies, this was my first attempt at winget using wingetcreate. I have closed the request, as I'm unsure how to fix this for now.

Reply:
No worries! Thanks for submitting your first PR. If you can find a direct link, feel free to open a new PR/reopen this one. For Battlenet specifically, it has had issues in the past as well:
- https://github.com/microsoft/winget-pkgs/pull/60225#pullrequestreview-971013280

Reply:
oh i see! I just read about the CDN and region stuff there. I did search beforehand but nothing for blizzard showed up, I forgot to check closed requests lol. Thanks for the info :)